apps/console/src/features/connections/resources/guides/swe/quick-start.tsx

```diff
@@ -210,33 +211,20 @@ const SIWEAuthenticationProviderQuickStart: FunctionComponent<SIWEAuthentication
                                 ".content"
                             }
                         >
-                            Go to <strong>Login Flow</strong> tab and click on <strong>Start with default
-                            configuration</strong>.
+                            Go to <strong>Login Flow</strong> tab and click on the <strong>Add Sign In Option</strong>
+                            button inside the login box. And select a Sign In With Ethereum connection.
```

Comment:
```suggestion
                            button inside the login box. And select a Ethereum connection.
```

Reply:
`Sign In With Ethereum`(https://login.xyz/) is the actual name of the service we are using here. `Ethereum` is the underneath blockchain.